src/bundles/SellerRegistration/components/ApplicationPreview/ApplicationPreview.js

```diff
@@ -23,6 +54,9 @@ const ApplicationPreview = ({ header, body, privateInfo, onClick }) => (
         <Link to="/review">Go back and edit</Link>
       </div>
     </div>}
+    <div>
+      <Changes {...body}/>
```

Comment:
Does this show up in the public view of the profile? You could use the same logic as PrivateInfo to hide {!onClick && privateInfo.disclosures